cds/config.py

```diff
@@ -146,6 +146,10 @@ def _parse_env_bool(var_name, default=None):
         # Every 12 minutes, not to be at the same time as the others
         "schedule": timedelta(minutes=12),
     },
+    "clean-tmp-videos": {
+        "task": "cds.modules.maintenance.tasks.clean_tmp_videos",
```

Comment:
Question: If I am not mistaken it shouldn't be any videos left in the tmp folder, I guess this is just to be extra safe in case something goes wrong, or am I missing something?

Reply:
Yes it is indeed! It covers the case when the extract metadata works, but extract frames, for some reasons, fails or never starts.